etc/kayobe/ansible/requirements.yml

```diff
@@ -1,7 +1,7 @@
 ---
 collections:
   - name: stackhpc.cephadm
-    version: 1.10.0
+    version: 1.12.1
```

Comment:
Is this required for a bug fix or something? Probably deserves a quick release note.

Reply:
Added small release note to reflect the bump in version.